### PR TITLE
Set ENABLE_OVS var when deploying knmstate with cluster-sync

### PR DESCRIPTION
**Pull request by rhrazdil**
Signed-off-by: Radim Hrazdil <rhrazdil@redhat.com>

**Is this a BUG FIX or a FEATURE ?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
> /kind enhancement

**What this PR does / why we need it**:

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by phoracek** [comment]
There would be no socket, Kubernetes will only mount an empty directory.

File: cluster/sync-operator.sh [hunk 1]
```diff
@@ -6,12 +6,28 @@ kubectl=./cluster/kubectl.sh
 
 source ./cluster/sync-common.sh
 
+function __yq() {
```
[the remark this answers is not in the document]
Comment (phoracek) [reply cluster/sync-operator.sh]: There would be no socket, Kubernetes will only mount an empty directory.

**Comment by qinqon** [comment]
/hold cancel
@rhrazdil has manually test it.

**Comment by rhrazdil** [comment]
We'll want to extend our e2e test with a scenario that creates ovs-bridge with primary nic connected to it as a port as well as a veth pair connecting linux-bride to the ovs-bridge.
However, this is currently blocked by an nmstate bug that causes removing ovs-bridge to fail (sometimes).
Let's include this scenario when https://bugzilla.redhat.com/show_bug.cgi?id=2012420 is resolved

**Comment by oshoval** [comment]
You might want please to change PR title, it is outdated comparing to the commit that was pushed as part of this PR
"Mount ovs socket unconditionally"

(just saw while trying to learn the flow)

**Pull request merged**